### DISCUSSION TITLE
nvim-tree is taking the buffer when closing a buffer

### DISCUSSION THREAD

**Discussion by VisionaryAppDev** [nvim-tree/nvim-tree.lua#2025] [Q&A]
Normally, when I type `:bd` an active buffer will close and switch to another one without any issue. But, when I have nvim-tree opened on the left side, it seems like nvim-tree is acting as a new buffer. And when I type `:bd` again, it will close nvim-tree and switch to another buffer with nvim-tree. Please check my reference video to understand what I mean.

https://user-images.githubusercontent.com/27724011/222088179-867e280b-1399-4a99-a749-e8b4b910014d.mp4

**Comment by gegoune** [comment]
That's not related to nvim tree. It's default vim/nvim behaviour. You can use some plugins like bufkill (or whatever they were called). There are issues opened about it but it's very unlikely that anything will be done about it.

**Reply by alex-courtis** [reply]
See #2014
